src/frontend/styles/main.css

```
@@ -0,0 +1,544 @@
+@import "tailwindcss/theme.css" layer(theme);
+/* Source paths for Tailwind to scan */
+@source "../../frontend";
+@source "../../../node_modules/@wedevs/plugin-ui/dist";
+
+.pui-root,
+.components-modal__screen-overlay {
+  @import "tailwindcss/preflight.css" layer(base);
+  @import "tailwindcss/utilities.css" layer(utilities) important;
+}
+
+@import '@wedevs/plugin-ui/styles.css';
```

Biome (2.4.4)

[error] This `@import` is in the wrong position.

(lint/correctness/noInvalidPositionAtImportRule)